packages/web/README.md

```diff
@@ -44,4 +44,48 @@ This code will make the transaction, let the user sign it using algosigner and s
 
 You can also use `web.sendTransaction()` or `web.signTransaction()` in a react app.
 
-**Note:** We don't support checkpoints yet. Currently `deployASA`, `deploySSC` functions don't work. User should directly pass assetIndex, appIndex instead of asaName, appName.
+**NOTE**:
+
+1. We don't support checkpoints yet. Currently `deployASA`, `deploySSC` functions don't work. User should directly pass assetIndex, appIndex instead of asaName, appName.
```

Comment:
```suggestion
1. We don't support checkpoints yet. Currently `deployASA`, `deployApp` functions don't work. User should directly pass assetIndex, appIndex instead of asaName, appName.
```